# Configuration message for computation units
# in_names: Names of topics to subscribe to
string[] in_names
# in_channels: Names of channels for incoming topics
string[] in_channels
# out_names: Names of topics to produce on
string[] out_names
# out_channels: Names of channels for outgoing topics
string[] out_channels
# manager: The host DyKnow manager
string manager